src/quote/swap/adapters/static/swap-data-config.ts

```diff
@@ -205,9 +199,7 @@ export const SwapDataConfig: {
           ],
           fees: [500],
           exchange: 3,
-          poolIds: [],
           pool: zeroAddress,
-          tickSpacing: [100],
```

Comment:
As ETH2X/ETH3X use Uniswap they shouldn't need tickSpacing here: https://github.com/IndexCoop/uniswap-quote-api/blob/main/swapConfig.json#L173. This still looks to give quotes most of the time but may be an issue sometimes?